packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-mobile.tsx

```diff
@@ -132,12 +185,12 @@ const Numbers = observer(
                     }}
                 />
             );
-            if (parseInt(value) < min || parseInt(selected_duration) > max) {
+            if (parseInt(value as string) < Number(min) || Math.trunc(selected_duration) > Number(max)) {
```

Comment:
As `selected_duration` is number (checked in console), `parseInt()` is not a proper method. That is why I replaced it with `Math.trunc()` as it do the same thing with numbers